wordpress-mobile/WordPress-iOS
# Show prepublish dialog in portrait only if title is present

Fixes #19336

## Description

On an iPhone, there isn't enough room to display both the keyboard and the publish dialog when in landscape. This change restricts the dialog to be portrait only, matching what Android currently does.

## Testing

To test:
- Launch on an iPhone (Creating a story post doesn't appear in iPad)
- Login if necessary
- Tap on the FAB '+'
- Tap on 'Story post' under 'Create New'
- Take a photo or select an existing one
- Tap on the arrow in the top right
- Rotate the device to landscape
- Verify it remains in portrait
- Close the dialog and view to get back to the home screen
- Tap on the FAB '+'
- Tap on 'Blog post' under 'Create New'
- Type something in the editor so the 'Publish' button gets enabled
- Tap on 'Publish' in the top right
- When the dialog appears, rotate the device to landscape
- Verify the dialog appears properly in landscape


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
